service/src/main/java/com/codeforcommunity/processor/ProtectedSiteProcessorImpl.java

```diff
@@ -876,6 +877,26 @@ && isAlreadyAdopted(siteId)
   }
 
   @Override
+
+  public List<SiteEntryImage> getUnapprovedImages(JWTData userData) {
+    assertAdminOrSuperAdmin(userData.getPrivilegeLevel());
+    List<SiteImagesRecord> imageRecords =
+            db.selectFrom(SITE_IMAGES).where(
+                    SITE_IMAGES.APPROVAL_STATUS.eq("SUBMITTED")).fetch();
```

Comment:
⛏️ Can we replace the raw "SUBMITTED" string with `ImageApprovalStatus.SUBMITTED.getApprovalStatus()` ? It's a bit longer, but this way, if we ever want to change how we represent an image pending approval, we can just change it once in the enum instead of changing a whole bunch of files.